Zend/zend_inheritance.c

```diff
@@ -1747,13 +1732,20 @@ static void do_interface_implementation(zend_class_entry *ce, zend_class_entry *
 	zend_function *func;
 	zend_string *key;
 	zend_class_constant *c;
+	uint32_t flags = ZEND_INHERITANCE_CHECK_PROTO | ZEND_INHERITANCE_CHECK_VISIBILITY;
+
+	if (!(ce->ce_flags & ZEND_ACC_INTERFACE)) {
+	 /* We are not setting the prototype of overridden interface methods because of abstract
```

Comment:
Nit: Indentation looks off.

Reply:
this comment moved without changes in indentation, but I agree.